apps/nextjs-example/components/transactionFlow/MultiAgent.tsx

```diff
@@ -23,7 +24,7 @@ export default function MultiAgentTransaction({
     useWallet();
 
   const [secondarySignerAccount, setSecondarySignerAccount] =
-    useState<Account>();
+    useState<Ed25519Account>();
```

Comment:
Is this change relevant to the PR?

Reply:
yes, newest sdk version removed the privateKey from Account